src/Raven.Client/Exceptions/Documents/Subscriptions/SubscriptionClosedException.cs

```diff
@@ -11,6 +11,7 @@ namespace Raven.Client.Exceptions.Documents.Subscriptions
     public class SubscriptionClosedException : SubscriptionException
     {
         internal bool CanReconnect { get; set; }
+        internal bool ClosedDueNoDocsLeft { get; set; }
```

Comment:
its SubscriptionClosedException, so 'close' is implied. Should be called 'NoDocsLeft' ?

Reply:
done